### DISCUSSION TITLE
Redirecting straight to another source/IdP

### DISCUSSION THREAD

**Discussion by RobinBeismann** [goauthentik/authentik#4449] [Q&A]
Hi,

I've figured out how to hide the normal login from the landing page, but is it possible to redirect straight to another IdP (Authentication Source as in Authentik Themes) instead of having to click it on the bottom?

I already went through the other discussions here but didn't find anything related.

Thanks,
Robin

**Comment by l4rm4nd** [comment]
Read this:

https://github.com/goauthentik/authentik/discussions/5470#discussioncomment-7143741

If there are no user fields and a single source for authentication only, you'll be instantly redirected to the source's login screen. No Authentik login screen anymore.
